src/commands/mod.rs

```diff
@@ -247,6 +247,19 @@ pub fn execute() -> Result<()> {
             .join(" ")
     );
 
+    #[cfg(feature = "profiling")]
+    let _guard = sentry::init((
+        env!("SENTRY_DSN").to_string(),
```

Comment:
```suggestion
        env!("SENTRY_DSN", "Environment variable `SENTRY_DSN` is required for `profiling` feature").to_string(),
```

Reply:
Its imo better than default `error: environment variable SENTRY_DSN not defined`